src/Validation/Validator.php

```diff
@@ -202,7 +202,7 @@ public function errors(array $data, bool $newRecord = true): array
 
             $canBeEmpty = $this->_canBeEmpty($field, $context);
 
-            $flags = static::EMPTY_ALL;
+            $flags = 0;
```

Comment:
What is 0?

Reply:
It has no special meaning except doing an logic `&` with it using any number will results in `0` itself.

Reply:
This is normal for a bitwise field.

Reply:
Should we have a constant for that to make its purpose more explicit? eg. `EMPTY_NULL`?

Reply:
Will do.

Reply:
Done